main/src/ca/uwaterloo/flix/language/phase/Simplifier.scala

```diff
@@ -48,13 +50,26 @@ object Simplifier {
       SimplifiedAst.Def(spec.ann, spec.mod, sym, fs, e, retType, eff, sym.loc)
   }
 
-  private def visitEffect(decl: MonoAst.Effect)(implicit flix: Flix, universe: Set[Symbol.EffectSym]): SimplifiedAst.Effect = decl match {
+  private def visitEffect(decl: MonoAst.Effect)(implicit flix: Flix, universe: Set[Symbol.EffectSym], root: MonoAst.Root): SimplifiedAst.Effect = decl match {
```

Comment:
I dont know why the arguments here are in the wrong order: I would put them:

universe, root, flix